site_source_files/content/blog/2020/moving-to-trunk-development.md

```diff
@@ -0,0 +1,19 @@
++++
+Description = "Why we are getting rid of `master` branch in favour of `trunk`"
+Title = "Moving to trunk"
+Date = 2020-07-01
+Author = "AutomatedTester"
+AuthorLink = "https://twitter.com/AutomatedTester"
+tags = ["selenium"]
+categories = ["general"]
++++
+
+Since the project started we have been following [trunk based development](https://trunkbaseddevelopment.com/). This was a very natural fit when we were using SVN over a decade ago on Google Code.
+
+As Google Code shut down we moved to GitHub and the git model of doing things. We moved there mostly due to the gravity that GitHub had created in Open Source projects.
+
+Unfortunately, it meant that we followed the standard use of `master` as our trunk to work off. Now that Github, and services that use Github, have improved support for non-master branches as default we are moving our default branch to `trunk`. It describes how we, as a project, work and is a more inclusive term.
```

Comment:
`s/Unfortunately,/This/` At the time it wasn't considered to be unfortunate, it's only with hindsight that this is true.